Resources/public/Admin.js

```diff
@@ -103,6 +103,31 @@ var Admin = {
                 checkboxClass: 'icheckbox_square-blue',
                 radioClass: 'iradio_square-blue'
             });
+
+            var previousIndex;
+
+            jQuery('tbody .iCheck-helper', subject).click(function (evt) {
```

Comment:
It should be possible to have an idea of what this function does without reading its code. Can you add some comments. Naming it might also be a good option, and would have the bonus of making stack traces that involve this function a bit more clear.